README.md

```diff
@@ -160,6 +160,26 @@ pytest tests/test_basics.py # Ces test peuvent prendre jusqu'à 60 secondes.
 ```
 :tada: OpenFisca-France est prêt à être utilisé !
 
+### Installez un environnement virtuel avec conda
+
+Nous conseillons cette procédure pour les personnes utilisant Windows et n'ayant pas d'environnement Python fonctionnel. Elle fonctionne également sous Linux et macOS.
+
+Ceci vous permet d'obtenir en une seule installation :
+- Python
+- Le gestionnaire de paquets [Anaconda.org](https://docs.anaconda.com/anacondaorg/user-guide/)
+- Le gestionnaire d'environnement Python virtuel : [conda](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html)
+
+Voici les étapes à suivre : 
+
+- Installer la version communautaire/gratuite en suivant la procédure décrite sur [le site Anaconda](https://www.anaconda.com/products/individual). A noter que Anaconda occupe beaucoup d'espace disque (>2 Go), vous pouvez installer à la place [Miniconda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/windows.html) qui occupe beaucoup moins d'espace disque. Cependant vous n'aurez pas l'interface graphique [Anaconda Navigator](https://docs.anaconda.com/anaconda/navigator/index.html) qui vous permet de gérer vos environnements. Si vous êtes habituée à gérer les choses en ligne de commande, préfèrer Miniconda.
+- Depuis le menu démarrer, exécuter `Anaconda Powershell Prompt`.
+- Exécuter les commandes suivantes dans le shell:
+  - Créer un environnement virtuel dédié : `conda create --name openfisca python=3.7`
+  - Activer l'environnement : `conda activate openfisca`
+  - Installer OpenFisca : `conda install -c openfisca.org openfisca-france`
```

Comment:
Mettre à jour le nom du channel pour éviter cette erreur ? : 
```
UnavailableInvalidChannel: The channel is not accessible or is invalid.
  channel name: openfisca.org
  channel url: https://conda.anaconda.org/openfisca.org
  error code: 404
```

```suggestion
  - Installer OpenFisca : `conda install -c openfisca openfisca-france`
```

Reply:
Malgré ce correctif, j'ai obtenu cette erreur mais peut-être est-elle due au démarrage ? 🤔 : 
```shell
conda install -c openfisca openfisca-france
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: \ 
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed                                                                                                                   

UnsatisfiableError: The following specifications were found to be incompatible with each other:

Output in format: Requested package -> Available versions
```

Reply:
Voici la commande vue ensemble @benoit-cty qui a permis de résoudre ce problème : 
`conda config --add channels conda-forge`

Par ailleurs, ajouter la commande pour sortir de l'environnement virtuel avec quelque chose comme ceci ? : 
> Lorsque vous avez achevé votre activité, vous pouvez quitter l'environnement virtuel avec la commande `conda deactivate`.

Enfin, pour ne pas avoir un environnement virtuel `base` qui s'ouvre à chaque nouveau terminal, nous avons configuré cette variable d'environnement (dans le `.bashrc`) : 
`export CONDA_AUTO_ACTIVATE_BASE=false`